modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -3439,9 +3440,11 @@ def _groupby_shuffle(
         else:
             obj = self
 
-        agg_func = functools.partial(
-            GroupByDefault.get_aggregation_method(how), func=agg_func
-        )
+        agg_method = GroupByDefault.get_aggregation_method(how)
+        original_agg_func = agg_func
+
+        def agg_func(grp, *args, **kwargs):
+            return agg_method(grp, original_agg_func, *args, **kwargs)
```

Comment:
this was changed in order to pass the `original_agg_func` as a second positional arg rather than a keyword arg so we can avoid the following error:

```python
# when we call 'df.groupby("a").fillna(0)' the keyword binding would resolve into something like:
>>> pandas.core.groupby.DataFrameGroupBy.aggregate(df.groupby("a"), 0, func="fillna")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: aggregate() got multiple values for argument 'func'
# instead we want the 'func' to be passed as a positional arg as well, so the fillna's arg
# would come after the func
>>> pandas.core.groupby.DataFrameGroupBy.aggregate(df.groupby("a"), "fillna", 0)
   b  c
0  3  a
1  2  b
2  5  c
```